### PR TITLE
The recent list doesn’t convert addresses to my nickname

### LINKED ISSUE #3817: The recent list doesn’t convert addresses to my nickname
**Describe the bug**
The recent list doesn’t convert addresses to my nickname, but when I select the address and it fills in the recipient field, the nickname shows up.

**Screenshots**
Image from iOS (1)
Image from iOS

**To Reproduce**
_Steps to reproduce the behavior_
1. Go to Wallet
2. Click on Send
3. Check Recents

**Expected behavior**
The expected behavior is to show nicknames in the drop-down list as well.

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Description**

The Recent list in the send flow now shows nicknames || name in the drop-down list.

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #3817

### CODE REVIEW

File: app/components/Views/SendFlow/AddressList/index.js [hunk 1]
```diff
@@ -227,6 +231,10 @@ class AddressList extends PureComponent {
 						<AddressElement
 							key={index}
 							address={address}
+							name={
```
Comment (andreahaku) [inline app/components/Views/SendFlow/AddressList/index.js:234]: `name={ identities[address]?.name) || networkAddressBook[address]?.name)}`

**Review by andreahaku — APPROVED** [review]
a part from the suggestion LGTM ;)

File: app/components/Views/SendFlow/AddressList/index.js (hunk 1 above)
Comment (blackdevelopa) [reply app/components/Views/SendFlow/AddressList/index.js]: Thanks. Now updated.

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮 🌮

**Pull request merged**